css/landing-page.css

```diff
@@ -186,6 +186,22 @@ header.masthead h1 {
     margin-left: 6rem;
   }
 }
+@media (min-width: 0px) and (max-width: 860px){
+  #page-content {
+    margin-left: 1rem;

```

Comment:
Why are you using rem here and not px or vw?

Reply:
Rems are more flexible than px imo - it allows the end user to keep their preferences in place and scale more easily.

Reply:
But wouldn't that end up in even less text when the font size increases since the margin is even larger? Though maybe for this one single left margin in the header it's fine.

Reply:
I suppose so, but since rem relies on the font-size of the root element, it doesn't seem to change as much. Also, I am aiming to have a decent amount of space between the left side of the screen and the start of the text - just seems more readable that way.

Reply:
Right I see. Thanks for clarification.